samples/core/output_a_directory/output_a_directory.py

```diff
@@ -0,0 +1,29 @@
+import kfp
```

Comment:
Needs the apache license blurb at the top of the file (as done here: https://github.com/kubeflow/pipelines/blob/master/samples/core/helloworld/hello_world.py)

Reply:
Done